ProyectoIntegrAccion.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "Proyecto IntegrAcción",
+  "type": "comunidad",
```

currently we only support `personal`/actualmente solo apoyamos `personal`:
```suggestion
  "type": "personal",
```